lib/write-pretty-file.js

```diff
@@ -23,10 +23,11 @@ export default async function writePrettyFile(path, content) {
 
   if (!parser) throw new Error(`Define parser for ${path}`);
 
-  await writeFile(
-    path,
-    pretier.format(content, {
+  const prettierContent = String(
```

Comment:
This line and variable doesn't need to be there, you can simply await prettier in the `writeFile()` call